### PR TITLE
feat(proposal): add counterpart bank account holder name

### PR DESCRIPTION
**Pull request by vkuprin**
Added new field bank account holder name for counterpart

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "dev:sdk-demo": "turbo run dev --filter=@team-monite/sdk-demo",
     "build": "turbo run build",
     "clean": "rm -rf packages/*/dist",
+    "clean:node": "find . -name node_modules -type d -prune -exec rm -rf '{}' + && find . -name .turbo -type d -prune -exec rm -rf '{}' + && find . -name dist -type d -prune -exec rm -rf '{}' +",
```
Comment (marcperezmonite) [inline package.json:21]: This doesn't seem related to this task, could you remove it?

Comment (vkuprin) [reply package.json:21]: Yes, I can keep it in separate PR

Comment (marcperezmonite) [reply package.json]: Better to keep them separated so we can properly track those later.

**Review by marcperezmonite — APPROVED** [review]
